scripts/lib/converter.rb

```diff
@@ -24,6 +24,9 @@ def each(&block)
   
   private
   
+  THUMBNAILS = 'thumbnails'
+  PROXIES = 'proxies'
+  
   def to_asset(xml)
```

Comment:
@mccalluc ah... i think i know now. Because you're parsing from JSON, which will end up as strings, and you'd rather remember a constant with a string value than the string value itself?